UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/FileContentHelper.cs

```diff
@@ -15,7 +15,6 @@ public static class FileContentHelper
         private const string FileContent_avcs = "AVCS";
         private const string FileContent_base = "Base";
         private const string FileContent_dvd = "Media','DVD_SERVICE'";
-
         private static TestConfiguration Config = new TestConfiguration();
         private static FssApiClient FssApiClient = new FssApiClient();
         public static async Task<string> CreateExchangeSetFile(HttpResponseMessage apiEssResponse, string FssJwtToken)
```

Comment:
downloadFileUrl .ToString() can be removed line no. 35

Reply:
As discussed, this is the existing ESS FT code used for small\medium update sets. Hence, we haven't changed that.

Thanks..